db/seeds.rb

```diff
@@ -99,7 +99,7 @@
 User.create(full_name: "Jeff Casimir", email: "demo+jeff@jumpstartlab.com", display_name: "j3", password: "password", role: 0)
 User.create(full_name: "Jorge Tellez", email: "demo+jorge@jumpstartlab.com", display_name: "novohispano", password: "password", role: 0)
 User.create(full_name: "Josh Cheek", email: "demo+josh@jumpstartlab.com", display_name: "josh", password: "password", role: 1)
-User.create(full_name: 'D G', email: "yourmom@gmail.com", display_name: 'Synergy', password: 'dg', role: 1)
+User.create(full_name: 'D G', email: "dg", display_name: 'Synergy', password: 'dg', role: 1)
```

Comment:
Line is too long. [92/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.